CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
   for `Cursor<&mut [u8]>`.
 - [[#256](https://github.com/rust-vmm/vm-memory/pull/256)] Implement `ReadVolatile`
   for `Cursor<T: AsRef[u8]>`.
+- [[#266](https://github.com/rust-vmm/vm-memory/pull/266)] Derive `Debug` for several
```

Comment:
v0.13.1 is already published, you should create a new version section for v0.14.0.

Reply:
Woops, fixed it now.